rightscript.go

```diff
@@ -433,7 +433,7 @@ func rightScriptIdByName(name string) (string, error) {
 	foundId := ""
 	for _, rs := range rightscripts {
 		// Recheck the name here, filter does a partial match and we need an exact one
-		if rs.Name == name && rs.Revision == 0 {
+		if rs.Name == name && rs.Revision == 0 && strings.Contains(rs.Lineage, strconv.Itoa(Config.Account.Id)) {
```

Comment:
Nice work! Thanks! But, I think that this could have false positives if the account ID happens to be a substring of the RightScript ID, such as in this made up lineage: `https://us-3.rightscale.com/api/acct/12345/right_scripts/123456789`. I think a more correct solution would to do a regexp match to capture the account ID from the lineage and then compare that to our configuration account ID, like: `"/api/acct/(\d+)/right_scripts/.+$"`.

Also, I fixed up our Travis CI config on master so it doesn't try to use that AWS secret when building for a fork so you can get your builds to actually pass if you rebase master into your branch.